forch/forchestrator.py

```diff
@@ -687,8 +694,12 @@ def get_sys_config(self, path, params):
         """Get overall config from faucet config file"""
         try:
             _, _, behavioral_config = self._get_faucet_config()
+            behavioral_config_map = {
+                'content': behavioral_config,
+                'warnings': dict(self._config_summary.warnings)
+            }
             reply = {
-                'faucet_behavioral': behavioral_config,
+                'faucet_behavioral': behavioral_config_map,
```

Comment:
This doesn't seem right and isn't logically consistent.  The warnings are not part of the behavioral config. Seems like the top-level should be faucet (parallel to forch) and then 'content' would be renamed to 'behavorial'.

Reply:
updated the output structure. PTAL

Reply:
Now it looks like:

- faucet
   * behavioral
   * structural
   * warnings
- forch
   * ...